src/store/modules/current-company/__tests__/reducer.test.js

```diff
@@ -0,0 +1,17 @@
+import currentCompanyReducer, {INITIAL_STATE} from 'store/modules/current-company/reducer';
+import {updateCompany} from 'store/modules/current-company/actions'
+
+describe('Current company reducer', () => {
+  it('returns unmodified state', () => {
+    expect(currentCompanyReducer(undefined, {})).toStrictEqual({ id: 0, name: '' })
+  })
+
+  it('handles SET_CURRENT_COMPANY', () => {
+    expect(
+      currentCompanyReducer(
+        INITIAL_STATE,
+        updateCompany({ id: 1, name: 'Company 1' })
+      )
+    ).toStrictEqual({ id: 1, name: 'Company 1' })
+  })
```

Comment:
Same as https://github.com/comarev/comarev-dashboard/pull/95#discussion_r906045456, this test is a bit hard to understand so there is room to improve, I'd do something like:
```suggestion
  it('handles SET_CURRENT_COMPANY', () => {
    const state = { id: 10, name: 'My Company' };
    const newState = { id: 11, name: 'Some other Company' };
    const expectedState = { id: 11, name: 'Some other Company' };

    const returnedState = currentCompanyReducer(state, updateCompany(newState));

    expect(returnedState).toStrictEqual(expectedState);
  });
```